NOLF2/ClientShellDLL/TO2/ScreenHostMission.cpp

```diff
@@ -147,7 +147,8 @@ LTBOOL CScreenHostMission::Build()
 
 
 	m_pDlg->SetBasePos(dlgPos);
-	m_pDlg->SetScale(g_pInterfaceResMgr->GetXRatio());
+	m_pDlg->ApplyPosition(g_pInterfaceResMgr->GetYRatio(), g_pInterfaceResMgr->Get4x3Offset());
+	m_pDlg->SetScale(g_pInterfaceResMgr->GetYRatio());
```

Comment:
again `CLTGUIWindow`'s SetScale function was renamed and `ApplyPosition` already calls the Parent `SetScale`